content/module-reference/processing-modules/filmic-rgb.md

```diff
@@ -89,33 +100,54 @@ dynamic range mapping
 
 ## scene
 
+---
+
+** Note:** the controls of the _scene_ tab are similar in principle to those of the
+_levels_ module (black, grey, white). The difference is _levels_ assume pixels
+values display-referred (between 0 and 100%), while _filmic_ allows to work on
+scene-referred pixels (between -infinity EV and + infinity EV), which forces
+to use a different interface.
+
+---
+
```

Comment:
We already have explanatory notes for the rest of the tabs so it makes sense to add one here but it doesn't really make sense to include as a note call-out at the start of the section so I'll change it to a standard paragraph.